wfs3/root.go

```diff
@@ -42,10 +42,10 @@ func Root(serveAddress string, checkOnly bool) (content *RootContent, contentId
 		return nil, contentId
 	}
 
-	apiUrl := fmt.Sprintf("http://%v/api", serveAddress)
-	conformanceUrl := fmt.Sprintf("http://%v/conformance", serveAddress)
-	collectionsUrl := fmt.Sprintf("http://%v/collections", serveAddress)
-	selfUrl := fmt.Sprintf("http://%v/", serveAddress)
+	apiUrl := fmt.Sprintf("%v/api", serveAddress)
```

Comment:
Per previous comment (go-wfs-config.toml) I don't think the protocol belongs in serveAddress.